content/routing.md

```diff
@@ -32,7 +32,7 @@ meteor add kadira:flow-router
 
 *Snippet: Adding the Flow Router package*
 
-At the time of writing this guide, Flow Router is at version 2.x.
+At the time of writing this guide, [Flow Router](https://github.com/kadirahq/flow-router) is at version 2.x.
```

Comment:
I think it's a good idea to link to original project, when we mention them in the guide.

So, here's the idea. I am think Meteor Guide is the place for show the direction. But it should not the place for all the content. It's technically possible to do it. 

And we can't mention all the features in all these projects here. I hope it's better to mention following types of links:

* Links to the original project 
* Some discussion on the Meteor Forum - Someone might wanna know the reason for removing auth from the router. We already discuss in on the Meteor Forum. So, we can link that.
* Some articles on Wikipedia and other